docs/platforms/javascript/common/apis.mdx

```diff
@@ -0,0 +1,942 @@
+              ],
+            },
+          },
+          {
+            name: "level",
+            type: '"fatal" | "error" | "warning" | "log" | "info" | "debug"',
+          },
+          {
+            name: "extra",
+            type: "Record<string, unknown>",
+            description:
+              "Additional data that should be sent with the exception.",
+          },
+          {
+            name: "tags",
+            type: "Record<string, string>",
+            description:
+              "Additional tags that should be sent with the exception.",
+          },
+          { name: "contexts", type: "Record<string, Record<string, unknown>>" },
+          { name: "fingerprint", type: "string[]" },
+        ],
+      },
+      description: "Optional additional data to attach to the Sentry event.",
+    },
+  ]}
+>
+  Capture an exception event and send it to Sentry. Note that you can pass not
+  only `Error` objects, but also other objects as `exception` - in that case,
+  the SDK will attempt to serialize the object for you, and the stack trace will
+  be generated by the SDK and may be less accurate.
+</SdkApi>
+
+<SdkApi
+name="captureMessage"
+signature={`function captureMessage(
+  message: string,
+  captureContext?: CaptureContext | SeverityLevel
+): EventId`}
+parameters={[
+  {
+    name: "message",
+    required: true,
+    type: "string",
+    description: "The message to capture.",
+  },
+  {
+    name: "captureContext",
+    type: {
+      name: "CaptureContext",
+      properties: [
+        {
+          name: "user",
+          type: {
+            name: "User",
+            properties: [
+              { name: "id", type: "string | number" },
+              { name: "email", type: "string" },
+              { name: "ip_address", type: "string" },
+              { name: "username", type: "string" },
+            ],
+          },
+        },
+        {
+          name: "level",
+          type: '"fatal" | "error" | "warning" | "log" | "info" | "debug"',
+        },
+        {
+          name: "extra",
+          type: "Record<string, unknown>",
+          description:
+            "Additional data that should be sent with the exception.",
+        },
+        {
+          name: "tags",
+          type: "Record<string, string>",
+          description:
+            "Additional tags that should be sent with the exception.",
+        },
+        { name: "contexts", type: "Record<string, Record<string, unknown>>" },
+        { name: "fingerprint", type: "string[]" },
+      ],
+    },
+    description: "Optional additional data to attach to the Sentry event.",
+  },
+]}
+>
+Capture a message event and send it to Sentry. Optionally, instead of a
+`CaptureContext`, you can also pass a `SeverityLevel` as second argument, e.g.
+`"error"` or `"warning"`.
+
+Messages show up as issues on your issue stream, with the message as the issue name.
+
+</SdkApi>
+
+## Enriching Events
+
+<SdkApi
```

Comment:
l: should we link to Enriching Events here? Similarly to my comment in scopes, we can also just link to specific sections in the respective methods

Reply:
this I would like to mostly get rid of eventually - most of the `enrichting events" pages are really just describing a single API or so!